lib/pymedphys/tests/dicom/test_dose.py

```diff
@@ -102,17 +102,35 @@ def test_require_patient_orientation():
 @pytest.mark.pydicom
 def test_sum_doses_in_datasets():
 
-    data1 = np.array([[[30, 20], [10, 5]], [[40, 25], [15, 8]]], dtype=np.uint32) * int(
-        1e6
-    )
-    scale1 = 1e-8
-
-    data2 = np.array(
-        [[[350, 400], [450, 475]], [[300, 375], [425, 460]]], dtype=np.uint32
-    ) * int(1e6)
-    scale2 = 2e-9
-
-    expected_sum = np.ones((2, 2, 2))
+    data1 = (
+        np.array(
+            [
+                [[0.90, 0.80, 0.70], [0.60, 0.50, 0.40]],
+                [[0.91, 0.81, 0.71], [0.61, 0.51, 0.41]],
+                [[0.92, 0.82, 0.72], [0.62, 0.52, 0.42]],
+                [[0.93, 0.83, 0.73], [0.63, 0.53, 0.43]],
+            ]
+        )
+        * int(1e2)
+    ).astype(np.uint32)
+
+    scale1 = 1e-2
```

Comment:
Nit: Define scale prior to data1, and then use the scale variable to adjust the data?

Reply:
Done :)